Calculator/calculator.py

```diff
@@ -0,0 +1,15 @@
+def sum(num1,num2):
+    # Check if the variable is an integer
```

Comment:
Is this ok? Are you only validating integer values? Can you change this to: 

`# Check if the input values are numbers`